website/frontend/src/assets/images/alert/error.svg

```diff
@@ -0,0 +1,11 @@
+<svg width="16" height="17" viewBox="0 0 16 17" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
could you try reduce icon sizes trough [SVGOMG](https://jakearchibald.github.io/svgomg/)?